routes/web.php

```diff
@@ -22,4 +22,5 @@
 Route::get('/home', 'HomeController@index')->name('home');
 Route::post('/post', 'HomeController@post')->name('post');
 Route::get('/users', 'UserController@users');
-Route::get('/user/{id}/home', 'UserController@user_home');
\ No newline at end of file
+Route::get('/user/{id}/home', 'UserController@user_home');
+Route::get('reset/mailable/send', 'SampleController@PasswordResetNotification');
```

Comment:
I think this line is not necessary. I do not see any code calling this route so I tried commenting it out and the functionality still works. Please confirm.